StripeFinancialConnections/StripeFinancialConnections/Source/Analytics/FinancialConnectionsSheetAnalytics.swift

```diff
@@ -50,6 +50,29 @@ struct FinancialConnectionsSheetFailedAnalytic: FinancialConnectionsSheetAnalyti
     let error: Error
 }
 
+/// Logged at the begining of the initial `synchronize` API call.
+struct FinancialConnectionsSheetInitialSynchronizeStarted: FinancialConnectionsSheetAnalytic {
+    let event = STPAnalyticEvent.financialConnectionsSheetInitialSynchronizeStarted
+    var clientSecret: String
+    var additionalParams: [String: Any] = [:]
+}
+
+/// Logged when the initial `synchronize` API call completes. Includes whether or not the call was a success, and the error otherwise.
+struct FinancialConnectionsSheetInitialSynchronizeCompleted: FinancialConnectionsSheetAnalytic {
+    let event = STPAnalyticEvent.financialConnectionsSheetInitialSynchronizeCompleted
+    var clientSecret: String
```

Comment:
if `var` is not needed, there's precedent to do `let` here:

https://github.com/stripe/stripe-ios/pull/3761/files#diff-a1d45f375cb5ca585cd6f3a0ef47b25cc99c5b4f3cc74518831ccbdcf0f916ffR35-R36